test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -163,6 +164,22 @@ public static ClientConfig buildClientConfig(URI controllerUri) {
 
     }
 
+    public static Map<String, Object> buildPatchedPravegaClusterSpec(String replicas, int replicaCount, String component, String namespace, String name, String kind) {
```

Comment:
Please, add comments describing this public method.

Reply:
done